### PR TITLE
Sync: Files missing to be in sync with source repo

### PR DESCRIPTION
**Pull request by fredericsimard**
This will sync only the files we want to be in sync with the source repo of gtfs.org.

### CODE REVIEW

File: scripts/simpleen-translator.sh
Comment (fredericsimard) [inline scripts/simpleen-translator.sh:1]: I wil have to resolve some conflicts in this file, nothing major.

File: scripts/fetchdata.sh
Comment (fredericsimard) [inline scripts/fetchdata.sh:1]: I've decided to copy this one over just to preserve it for future reference. It will not be active once the fork is merged back into the origin repo.

**Review by Sergiodero — APPROVED** [review]
Thanks Fred, I'm approving with the understanding that this is part of the set up needed for translations, as well as moving the fetch script for reference.

**Comment by fredericsimard** [comment]
@Sergiodero Thanks. I'd still like to have another pair of eyes on this, just to make sure, like @tzujenchanmbd, at least. If you don't mind.

**Pull request closed without merging**
